go.sum

```diff
@@ -1,5 +1,7 @@
 github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/evrenios/slack v0.6.7 h1:Owaw4BdxojykM9pdyLHPo6MChqJq06yFon/28nRrcao=
```

Comment:
Could you remove these references to your fork please. `go mod tidy` should do it.

Reply:
just ran in @ollieparsley